src/global.ts

```diff
@@ -348,6 +348,10 @@ declare global {
         }
     }
 
+    interface Window {
+        AutomaticBonusProgression: typeof AutomaticBonusProgression;
+    }
+
```

Comment:
Why is it throwing an error now and not before?

Reply:
Good question. I didn't see the `const AutomaticaBonusProgression: typeof ABP` a few lines above and assumed that the definition was accidentally removed.

```ts
  // hooks/load.ts line 172+173

  // Make available immediately on load for module subclassing
  window.AutomaticBonusProgression = AutomaticBonusProgression;
```
This throws a type error now: `Property 'AutomaticBonusProgression' does not exist on type 'Window & typeof globalThis'.ts(2339)`

Reply:
Ok that was wrong. I forgot that the repo now defaults to `v13-dev`. The interface does exist in master and was removed somewhere in the v13 dev cycle.